src/snowcli/utils.py

```diff
@@ -578,6 +576,26 @@ def create_project_template(template_name: str, project_directory: str | None =
     )
 
 
+def sql_to_python_return_type_mapper(resource_return_type: str) -> str:
```

Comment:
Is this a copy of `_sql_to_python_return_type_mapper` from snowpark/common? I think `_sql_to_python_return_type_mapper` should be replaced by your function